### DISCUSSION TITLE
tui-input-time

### DISCUSSION THREAD

**Discussion by ivan1fun** [taiga-family/taiga-ui#2659] [Q&A] [answered]
Привет!
Пытаюсь использовать в компоненте `TuiInputTimeComponent` время в формате `MM:SS`. Такой формат не предусмотрен. Можно ли как-то его внедрить?

**Comment by waterplea** [comment, marked as answer]
Привет. Сейчас это никак не сделать, заведи, пожалуйста, задачу на доработку.
